graal-js/src/com.oracle.truffle.js/src/com/oracle/truffle/js/builtins/GlobalBuiltins.java

```diff
@@ -173,6 +179,10 @@ protected GlobalBuiltins() {
         decodeURIComponent(1),
         eval(1),
 
+        // ModuleBlock methods
+        serialize(1),
```

Comment:
I have not checked the spec, but find it weird that this is a global method. That means the user is able to call `serialize(...)` in any JavaScript code? It that defined by the spec, or is that your test code?

Reply:
I think that `serialize`/`deserialize` here are not part of the module block spec, but are rather "our" (experimentaal) builtins to allow dynamic allocation of them (a browser would have them as part of e.g. `postMessage()` when a module block is passed to a web worker, or similar in Node.js) -- hence, they should at least be disabled by default and enabled only under an experimental flag

Reply:
Added the Specialization annotation but couldn't find a guard variable for an experimental jsContext. Does it need to be created or did I overlook it?

Reply:
I think that we should just introduce an option (called e.g. `experimental-module-block-serialization-api`) in [`JSContextOptions`](https://github.com/oracle/graaljs/blob/7d822189adf3dcd8c8cacff4fcdc59d61fae202e/graal-js/src/com.oracle.truffle.js/src/com/oracle/truffle/js/runtime/JSContextOptions.java#L67), and disable it by default. Let's discuss the details in a call.

Reply:
Introduced the additional option.